browser/components/molecules/MyRequestedPrrrs/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react'
+import moment from 'moment'
+import Link from '../../atoms/Link'
+import Date from '../../atoms/Date'
+import Button from '../../atoms/Button'
+import GithubUsername from '../../atoms/GithubUsername'
+import PrrrsTable from '../PrrrsTable'
+import ErrorMessage from '../../atoms/ErrorMessage'
+
+export default class MyRequestedPrrrs extends Component {
+  static propTypes = {
+    currentUser: PropTypes.object.isRequired,
+    prrrs: PropTypes.array.isRequired,
+  }
+
+  constructor(props){
+    super(props)
+    this.state = {error: null}
+  }
+  renderAdditionalHeaders(){
+    return [
+      <th key="claimed">Claimed By</th>,
+      <th key="completed">Completed</th>,
+      <th key="actions">Actions</th>
+    ]
+  }
+  renderAdditionalCells = (prrr) => {
+    const { currentUser } = this.props
+    if ( prrr.claimed_by &&  !prrr.completed_at ) {
+      return [
+        <td key="claimed">
```

Comment:
try refactoring this into 3 methods. this one and `this.renderClaimedCell` and `this.renderCompletedCell`